requirements.txt

```diff
@@ -0,0 +1,4 @@
+dgl
+torch
+matplotlib
+
```

Comment:
Note: rdkit is listed in travis yml install script and in conda environment yml but not in requirements txt